test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2782,14 +2782,32 @@ def forward(self, x):
         self.run_test(Zero_(), x)
 
     @skipIfUnsupportedMinOpsetVersion(9)
-    def test_new_zero(self):
+    def test_new_zeros(self):
```

Comment:
this test is modified in my update slice PR, isn't it better to write another test?

Reply:
I'm trying to test the dtype argument in the symbolic. I can rebase the PR and update it once your PR is merged.